specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/streams/kafka/subscribe.expire.message/server.rpt

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+property timestamp ${kafka:timestamp()}
+
+accept "zilla://streams/kafka0"
+        option zilla:window 8192
+        option zilla:transmission "duplex"
+
+accepted
+
+read zilla:begin.ext ${kafka:matchBeginEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .capabilities("FETCH_ONLY")
+                                .topic("mqtt-messages")
+                                .filter()
+                                    .headers("zilla:filter")
+                                        .sequence("sensor")
+                                        .sequence("one")
+                                        .build()
+                                    .headerNot("zilla:qos", "1")
+                                    .headerNot("zilla:qos", "2")
+                                    .build()
+                                .evaluation("EAGER")
+                                .build()
+                            .build()}
+
+write zilla:begin.ext ${kafka:beginEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .capabilities("FETCH_ONLY")
+                                .topic("mqtt-messages")
+                                .partition(0, 0, 1, 1)
+                                .build()
+                            .build()}
+
+connected
+
+write await SEND_DATA
+write zilla:data.ext ${kafka:dataEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .fetch()
+                                    .timestamp(timestamp)
+                                    .filters(1)
+                                    .partition(0, 2, 2)
+                                    .progress(0, 3)
+                                    .progress(1, 1)
+                                    .key("sensor/one")
+                                    .header("zilla:filter", "sensor")
+                                    .header("zilla:filter", "one")
+                                    .header("zilla:local", "client")
+                                    .headerInt("zilla:timeout-ms", 1000)
```

Comment:
We've been using `mqtt` naming concepts such as `zilla:filter` (topic filter), `zilla:local` (no local), `zilla:format` (format), so it seems like we should name this `zilla:expires` or similar (expiry interval) instead of timeout.

Also, per spec "If present, the Four Byte value is the lifetime of the Application Message in seconds" so seems like the unit should be seconds not milliseconds, to align with the mqtt expiry interval concept.

Reply:
When we initially discussed this we agreed that we should stick to Zilla terminology, not the mqtt ones. Also I'm not sure what it was but there was a reason that we changed the timeout to ms. Maybe because in Kafka we use milliseconds?

Reply:
Let me know if we should change this.

Reply:
Yes, we should change `zilla:timeout-ms` to `zilla:expiry` please.

These header names are all derived from `mqtt` specific concepts except `timeout-ms` which sticks out as being inconsistent at the moment.